lms/templates/courseware/progress_graph.js

```diff
@@ -103,18 +131,74 @@ $(function () {
     descending_grades = sorted(grade_cutoffs, key=lambda x: grade_cutoffs[x], reverse=True)
     for grade in descending_grades:
         percent = grade_cutoffs[grade]
+        ## safe-lint: disable=javascript-jquery-append
```

Comment:
IMO safelint is working correctly in catching these, we should be using HtmlUtils for this kind of operation